launchers/install-full-deps

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sudo apt install -yy gstreamer1.0-vaapi gstreamer1.0-tools vainfo mpv python3-tinydb python3-jinja2 \
+                     wl-clipboard gnome-screensaver gnome-screenshot ffmpeg
+
+# kivu team PPA
+sudo apt-add-repository -y 'ppa:kivuteam/ppa'
+sudo apt update
+
+# ydotool
+sudo apt install -yy ydotool ydotoold
```

Comment:
Is there a reason you use -yy on the installs? The [docs](https://manpages.ubuntu.com/manpages/xenial/man8/apt-get.8.html)  only show -y as a flag, and this installs on my system without the -yy

Reply:
that is a typo from an old habit, thanks